### PR TITLE
Add note about minimum Kuryr version for OSP13z13

### PR DESCRIPTION
**Pull request by dulek**

### CODE REVIEW

**Comment by dulek** [comment]
@luis5tb

**Comment by dulek** [comment]
@maxwelldb

File: install_config/topics/ocp_osp_sdn.adoc [hunk 1]
```diff
@@ -36,3 +36,9 @@ To enable Kuryr SDN, your environment must meet the following requirements:
 * Neutron Trunk ports extension enabled
 * If ML2/OVS Neutron driver is used the OpenvSwitch firewall driver must be
 used, instead of the ovs-hybrid one.
+
+[IMPORTANT]
+====
+To use Kuryr with {rhosp} 13z13 you need to use version v3.11.306 or higher of
+Kuryr container images.
+====
```
Comment (maxwelldb) [inline install_config/topics/ocp_osp_sdn.adoc:44]:
```suggestion
[IMPORTANT]
====
To use Kuryr with {rh-openstack-first} 13.0.13, the Kuryr container images must be version v3.11.306 or higher.
====
```
The MM.zz pattern doesn't seem like it's really used in OCP or OSP documentation. Where'd you see it originally, and do you agree with my substitution?

**Review by maxwelldb — CHANGES_REQUESTED** [review]

File: install_config/topics/ocp_osp_sdn.adoc (hunk 1 above)
Comment (dulek) [reply install_config/topics/ocp_osp_sdn.adoc:44]: You're right, I now see that we only use 13z13 internally and according to https://bugzilla.redhat.com/show_bug.cgi?id=1873179 13.0.13 is the correct notation.

File: install_config/topics/ocp_osp_sdn.adoc [hunk 2]
```diff
@@ -36,3 +36,9 @@ To enable Kuryr SDN, your environment must meet the following requirements:
 * Neutron Trunk ports extension enabled
 * If ML2/OVS Neutron driver is used the OpenvSwitch firewall driver must be
 used, instead of the ovs-hybrid one.
+
+[IMPORTANT]
+====
+To use Kuryr with {rh-openstack-first} 13.0.13, the Kuryr container images must
+be version v3.11.306 or higher.
```
Comment (maxwelldb) [inline install_config/topics/ocp_osp_sdn.adoc:43]:
```suggestion
be version 3.11.306 or higher.
```
One last thing. @dulek

**Comment by maxwelldb** [comment]
And two housekeeping questions:

1. This is only for the 3.11 OCP docs, right?
2. Is there a specific BZ or Jira ticket that this is tied to?

**Review by maxwelldb — APPROVED** [review]

**Comment by maxwelldb** [comment]
@openshift/team-documentation PTAL. TY!

File: install_config/topics/ocp_osp_sdn.adoc [hunk 3]
```diff
@@ -36,3 +36,9 @@ To enable Kuryr SDN, your environment must meet the following requirements:
 * Neutron Trunk ports extension enabled
 * If ML2/OVS Neutron driver is used the OpenvSwitch firewall driver must be
 used, instead of the ovs-hybrid one.
+
+[IMPORTANT]
+====
+To use Kuryr with {rh-openstack-first} 13.0.13, the Kuryr container images must
```
Comment (maxwelldb) [inline install_config/topics/ocp_osp_sdn.adoc:42]:
```suggestion
To use Kuryr with OpenStack 13.0.13, the Kuryr container images must
```
The 3.11 docs apparently don't use the OSP common attributes, yet. Using the old style for now.

**Comment by dulek** [comment]
Yes, this is just for 3.11, 4.x is fixed from the start. There's no BZ, there was a customer case but we consider it user error to upgrade OSP without touching OCP.

**Comment by maxwelldb** [comment]
@itzikb-redhat Are you able to confirm this note? Thanks!

**Comment by itzikb-redhat** [comment]
ok from my side

**Comment by maxwelldb** [comment]
@itzikb-redhat TY!

**Comment by bmcelvee** [comment]
LGTM

**Comment by maxwelldb** [comment]
Thanks!

**Pull request merged**